### PR TITLE
Display all users

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @users = User.page(params[:page]).per 10
```
Comment (zenith19) [inline app/controllers/users_controller.rb]: @users.all.page(params[:page]).per 10

**Comment by sujoydatta** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
OK

**Comment by MuhammadTamzid** [comment]
Ok

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @users = @users.all.page(params[:page]).per 10
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: no need `all`

File: lib/tasks/_shared.rake [hunk 3]
```diff
@@ -28,18 +28,13 @@ def process_category_task name
...
-        author: author, publish_date: publish_date, page: page, rating: rating
-      puts "#{obj.class.name} has been created!"
-      obj.attributes.each {|_name, value| puts "#{_name}: #{value}"}
-    rescue => exception
-      _dump_errors exception
-    end
-  else
-    puts "Error: Argument conflict"
+  begin
+    obj = Book.create! category_id: category_id, title: title,
```
Comment (dieunb) [inline lib/tasks/_shared.rake]: fabricator gem to fake data

File: lib/tasks/user_book.rake [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
     end
   end
 end
 
-def process_user_book_task user_id, book_id, favourite_status, study_status
+def process_user_book_task user_id, book_id, favourite, status
   begin
-    obj = Like.create! user_id: user_id, book_id: book_id,
-      favourite_status: favourite_status, study_status: study_status
+    obj = UserBook.create! user_id: user_id, book_id: book_id,
```
Comment (dieunb) [inline lib/tasks/user_book.rake]: fabricator gem to fake data

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (MuhammadTamzid) [inline app/controllers/users_controller.rb]: @users = @users.page(params[:page]).per 10

**Pull request merged**